tests/test_meta.py

```diff
@@ -523,6 +524,15 @@ def test_is_literal():
     assert not is_literal([1, 2, 3])
 
 
+def test_is_literal_py39(mocker):
```

Comment:
I meant for it to run the same checks as `test_is_literal()` with the version flags to test the updated condition.

It didn't look like the branch was being tested on its own ([previous](https://coveralls.io/jobs/119565620/source_files/6749706810#L364) vs. [current PR coverage for posix-3.11](https://coveralls.io/jobs/119671140/source_files/6815246843#L365)):

https://github.com/Fatal1ty/mashumaro/blob/0f10329865813abb9a40616f154be2c39b47bdac/mashumaro/core/meta/helpers.py#L367-L368

I just noticed the branch gets tested through the version matrix on the CI ([combined coverage](https://coveralls.io/builds/59207112/source?filename=mashumaro%2Fcore%2Fmeta%2Fhelpers.py#L364)).

I can remove the test if you'd like. It really only makes sense to have it in the context of local development since the CI matrix already tests the logic.

Reply:
Yep, The implementation of Python types varies from version to version, so we usually can't simply pretend that we're on a different version without pain. That's why I'm in favor of running tests on the real different versions of Python. I think we should remove this test and let CI do the job :)

Reply:
Done!